test/integration/smoke/test_ssvm.py

```diff
@@ -1197,3 +1205,148 @@ def test_10_destroy_cpvm(self):
         # Call to verify cloud process is running
         self.test_04_cpvm_internals()
         return
+
+    @attr(
+        tags=[
+            "advanced",
+            "advancedns",
+            "smoke",
+            "basic",
+            "sg"],
+        required_hardware="false")
```

Comment:
Is it possible to run this test on Simulator? If not please change required_hardware to true.